packages/plainjs/jest.config.ts

```diff
@@ -1,11 +1,17 @@
-import { JestConfigWithTsJest } from 'ts-jest';
+import type { JestConfigWithTsJest } from 'ts-jest';
 
 const jestConfig: JestConfigWithTsJest = {
-  preset: 'ts-jest/presets/default-esm', // or other ESM presets
+  extensionsToTreatAsEsm: ['.ts'],
```

Comment:
Tests were failing in this package, because the `extensionsToTreatAsEsm` option was missing. I simply copied the options to make it similar to other files.